packages/web/src/scss/components/Grid/_theme.scss

```diff
@@ -1,4 +1,16 @@
 @use '@tokens' as tokens;
+@use '../../settings/dictionaries';
+
+$alignments-x: (
```

Comment:
the default is acutally called `normal`, so please reflect it in the demo (and docs I assume) and add a `stretch` variant to both.